install-deps.sh

```diff
@@ -245,12 +311,16 @@ else
             ensure_decent_gcc_on_rh $dts_ver
 	fi
         ! grep -q -i error: $DIR/yum-builddep.out || exit 1
+        $SUDO $yumdnf install -y python-coverage python-tox
+        ensure_min_npm_version "$yumdnf install -y npm"
         ;;
     opensuse|suse|sles)
         echo "Using zypper to install dependencies"
         $SUDO zypper --gpg-auto-import-keys --non-interactive install lsb-release systemd-rpm-macros
         munge_ceph_spec_in $DIR/ceph.spec
         $SUDO zypper --non-interactive install $(rpmspec -q --buildrequires $DIR/ceph.spec) || exit 1
+        $SUDO zypper --non-interactive install python-bcrypt python-coverage python-tox
+        ensure_min_npm_version "zypper --non-interactive install npm"
```

Comment:
Actually, the full form of the command should be `zypper --gpg-auto-import-keys --non-interactive install --no-recommends npm` (sorry to say)

Reply:
I have an upstream PR pending to unify this: https://github.com/ceph/ceph/pull/20434

Obviously this change conflicts with that one, but the conflict won't be hard to resolve.

Reply:
Yes, I'll resolve the conflicts when ceph#20434 is merged